src/server/dflycmd.cc

```diff
@@ -328,22 +382,20 @@ void DflyCmd::FullSyncFb(FlowInfo* flow) {
     return;
   }
 
-  if (saver->Mode() != SaveMode::SUMMARY) {
-    // TODO: we should be able to stop earlier if requested.
-    ec = saver->SaveBody(nullptr);
-    if (ec) {
-      LOG(ERROR) << ec;
-      return;
-    }
+  // TODO: we should be able to stop earlier if requested.
```

Comment:
can be an unrelated PR: redis stops writing into a rdb file upon exit after a second sigterm but we do not. 

at least worth opening an issue and then someone can do it unrelated to replication.

Reply:
I'll be wiring cancellation into the rdb saver anyways, so I can add it there